libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -478,6 +499,31 @@ bool AP_Arming::manual_transmitter_checks(bool report)
     return true;
 }
 
+bool AP_Arming::servo_checks(bool report)
+{
+    bool check_passed = true;
+    for (uint8_t i = 0; i < NUM_SERVO_CHANNELS; i++) {
+        SRV_Channel *ch = SRV_Channels::srv_channel(i);
+        if (ch != nullptr && ch->get_function() != SRV_Channel::k_none) {
```

Comment:
Early-`continue`?

I'd also break the thing up into two separate if-clauses, but that's more a personal-preference-thing that seems to be indulged...